src/com/layout/grid/grid.less

```diff
@@ -0,0 +1,19 @@
+.container {
+  &.-grid {
+    flex-wrap: wrap;
+    flex-direction: row;
+
+    justify-content: flex-start; /* space-between; */
+    align-items: flex-start;
+    margin: -0.5rem;	/* to undo the effect of the 0.5rem margin in items */
+
+    & .-col {
+      padding: 0.5rem;
+      box-sizing: border-box;
+    }
+  }
+
+  &.-selectable {
+    padding-top: 0.75rem;
+  }
```

Comment:
Why would a container that can be selected have padding on top?

Reply:
This is more an issue of actual naming than anything. I was struggling to think of another name for a component that had a `Grid` + a `GridSelector`.

Although I did just realise that it's meant to be only if it's both `-grid` and `-selectable` so I'll fix that.

Reply:
Cool. Yeah appears to be the core of the issue. What makes a `GridSelector` something that selects isn't exactly clear.